seatunnel-e2e/seatunnel-connector-v2-e2e/connector-clickhouse-e2e/src/test/resources/fake_to_clickhouse_with_multi_table.conf

```
@@ -0,0 +1,114 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+######
+###### This config file is a demonstration of batch processing in seatunnel config
+######
+
+env {
+  parallelism = 1
+  job.mode = "BATCH"
+  job.name = "fake_to_clickhouse_with_multi_table"
+}
+
+source {
+  FakeSource {
+    tables_configs = [
+      {
+        schema = {
+          table = "sink_table1"
+          fields {
+            c_string = string
+            c_boolean = boolean
+            c_tinyint = tinyint
+            c_smallint = smallint
+            c_int = int
+            c_bigint = bigint
+            c_float = float
+            c_double = double
+            c_decimal = "decimal(30, 8)"
+            c_date = date
+            c_time = time
+            c_timestamp = timestamp
+            c_map = "map<string, int>"
+            c_array = "array<int>"
+          }
+          primaryKey {
+            name = "c_string"
+            columnNames = [c_string]
+          }
+        }
+        row.num = 5
+      },
+      {
+        schema = {
+          table = "sink_table2"
+          fields {
+            c_string = string
+            c_boolean = boolean
+            c_tinyint = tinyint
+            c_smallint = smallint
+            c_int = int
+            c_bigint = bigint
+            c_float = float
+            c_double = double
+            c_decimal = "decimal(30, 8)"
+            c_date = date
+            c_time = time
+            c_timestamp = timestamp
+            c_map = "map<string, int>"
+            c_array = "array<int>"
+          }
+          primaryKey {
+            name = "c_string"
+            columnNames = [c_string]
+          }
+        }
+        row.num = 6
+      }
+    ]
+    plugin_output = "sink_multi_table"
+  }
+}
+
+sink {
+  Clickhouse {
+    plugin_input = "sink_multi_table"
+    host = "clickhouse:8123"
+    database = "default"
+    table = "sink_multi_table"
```

```suggestion
    table = "${table_name}"
```
we use placeholder to make sure one sink config can write into multi table.

---

Thanks, got it. I’ll use this approach to revise the test cases and documentation.